basis/http/http.factor

```diff
@@ -152,6 +159,7 @@ redirects ;
         <url>
             H{ } clone >>query
         >>url
+        <url> >>proxy-url
```

Comment:
The <request> word also creates an empty url with an empty query and throws that away in <client-request> (ie all words except calling http-request on a manually created request). So I think it's fairly consistent with the rest of the implementation.